File: src/Backend/Common/Models/Seasons/Supply.cs
using System;

namespace Company.Common.Models.Seasons
{
    public class Supply
    {
        public int Id { get; set; }

        public DateTime? DeleteTime { get; set; }

        public int? DeleterId { get; set; }

        public int LogisticId { get; set; }

        public Logistic Logistic { get; set; }

        public int BatchesCount { get; set; }

        public float RiskCoefficient { get; set; }

        public DateTime DeliveryDate { get; set; }

        public DateTime FabricDate { get; set; }

        public int TransportCost { get; set; }

        public int BrokerCost { get; set; }

        public int WtsCost { get; set; }

        /// <summary>
        /// у.е.
        /// </summary>
        public int Other { get; set; }
    }
}

File: src/Frontend/projects/core/src/lib/models/dto/seasons/supply.ts
export class Supply {
    public id: number;
    public batchesCount: number;
    /** [%, <100] */
    public riskCoefficient: number;
    public deliveryDate: Date;
    public fabricDate: Date;
    /** [у.е.] */
    public brokerCost: number;
    /** СВХ [у.е.] */
    public wtsCost: number;
    /** [у.е.] */
    public transportCost: number;
    /** [у.е.] */
    public other: number;

    static fromJSON(obj: any) {
        if (!obj) {
            return null;
        }
        return Object.assign(new Supply(), obj,
            {
                deliveryDate: obj.deliveryDate ? new Date(obj.deliveryDate) : null,
                fabricDate: obj.fabricDate ? new Date(obj.fabricDate) : null,
            });
    }
}
